<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KinectWidget</class>
 <widget class="QWidget" name="KinectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBox_pipeline">
     <item>
      <property name="text">
       <string>Cpu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OpenGL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OpenCL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cuda</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_Close">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="TransformationWidget" name="myTransformationWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_Open">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox_KinectSerials"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_Grab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Grab</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TransformationWidget</class>
   <extends>QWidget</extends>
   <header>TransformationWidget.h</header>
   <container>1</container>
   <slots>
    <signal>matrixchanged(Transform)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Open</sender>
   <signal>clicked()</signal>
   <receiver>KinectWidget</receiver>
   <slot>OpenKinect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Close</sender>
   <signal>clicked()</signal>
   <receiver>KinectWidget</receiver>
   <slot>CloseKinect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Grab</sender>
   <signal>clicked()</signal>
   <receiver>KinectWidget</receiver>
   <slot>GrabKinect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>myTransformationWidget</sender>
   <signal>matrixchanged(Transform)</signal>
   <receiver>KinectWidget</receiver>
   <slot>TransformationChanged(Transform)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenKinect()</slot>
  <slot>CloseKinect()</slot>
  <slot>GrabKinect()</slot>
  <slot>TransformationChanged(Transform)</slot>
 </slots>
</ui>
